packages/esm-service-queues-app/src/queue-screen/queue-screen.component.tsx

```diff
@@ -19,13 +73,6 @@ const QueueScreen: React.FC<QueueScreenProps> = () => {
     return <div>Error</div>;
```

Comment:
We need a better error state here. Consider using the `ErrorState` component from the framework:

```tsx
 return <ErrorState error={error} headerTitle={t('queueScreenError', 'Queue screen error')} />;
```